pkg/catalog/types.go

```diff
@@ -42,6 +43,23 @@ type nodegroup struct {
 	ConfigMeta          map[string]string `json:"config_metadata"`
 }
 
+type connections struct {
+	Clusters   map[string]clusterConnections           `json:"clusters"`
+	ConfigMaps map[string]map[string]map[string]string `json:"configmaps"`
+}
+
+type clusterConnections struct {
+	Version    string                  `json:"version"`
+	Services   map[string]ngRefkeysMap `json:"services"`
+	Nodegroups map[string]nodegroup    `json:"nodegroups"`
+	Distros    map[string]refkeysMap   `json:"distros"`
+	Cluster    cluster                 `json:"cluster"`
```

Comment:
The Cluster object already has Name, Isolated, ID, and ConfigMeta in it, so this seems redundant.

This seems like it should be exactly like the configmeta type above, except we would always leave Connections empty. Do we need to define these as two different types?